.github/workflows/ci.yaml

```diff
@@ -41,4 +41,4 @@ jobs:
           version: nightly
 
       - name: Run tests
-        run: FOUNDRY_PROFILE=ci forge test
\ No newline at end of file
+        run: make test
```

Comment:
CI needs to be configured with an ETH_RPC_URL env var, see [here](https://github.com/foundry-rs/foundry/blob/911a76e46753fd266cec182d17debb5429d88c9f/.github/workflows/test.yml#L33-L34) for an example of how to configure this

Reply:
Thanks for the example! added here: https://github.com/ScopeLift/umbra-protocol/pull/341/commits/c1ad588dbc2dd150818f714ca371f0d0ea0f8578